### DISCUSSION TITLE
Fresh PySDM install not working on Window 10

### DISCUSSION THREAD

**Discussion by ghost** [open-atmos/PySDM#471] [Q&A] [answered]
I used "git for windows" to download and install PySDM via command line. However, it is not even moving past the import statements. After typing in the console, 

`from PySDM.physics import si`

nothing happens. It is not throwing error. Any ideas how can this be fixed.

**Comment by slayoo** [comment, marked as answer]
Hello,
How exaclty have you installed PySDM?
If you clone with git, the best way to install PySDM is to do "pip install -e" (see https://stackoverflow.com/questions/42609943/what-is-the-use-case-for-pip-install-e).
Sylwester

**Reply by ghost** [reply]
I simply have used,
`pip install git+https://github.com/atmos-cloud-sim-uj/PySDM.git`
as described in the readme file.

**Reply by ghost** [reply]
do you mean I should clone repo?

**Reply by slayoo** [reply]
OK, ``from PySDM.physics import si`` should not output anything - do you mean Python hangs here?

**Reply by ghost** [reply]
yes, the pointer keep blinking. so I guess you are right python hangs

**Reply by ghost** [reply]
ok I tried installing again, and this time `from PySDM.physics import si` gave the following error.

> Traceback (most recent call last):
>   File "<stdin>", line 1, in <module>
>   File "C:\Program Files\Python39\lib\site-packages\PySDM\__init__.py", line 8, in <module>
>     from .builder import Builder
>   File "C:\Program Files\Python39\lib\site-packages\PySDM\builder.py", line 11, in <module>
>     from PySDM.attributes.impl.mapper import get_class as attr_class
>   File "C:\Program Files\Python39\lib\site-packages\PySDM\attributes\impl\mapper.py", line 10, in <module>
>     from PySDM.attributes.physics.terminal_velocity.terminal_velocity import TerminalVelocity
>   File "C:\Program Files\Python39\lib\site-packages\PySDM\attributes\physics\terminal_velocity\terminal_velocity.py", line 6, in <module>
>     from .gunn_and_kinzer import Interpolation
>   File "C:\Program Files\Python39\lib\site-packages\PySDM\attributes\physics\terminal_velocity\gunn_and_kinzer.py", line 6, in <module>
>     from scipy.interpolate import Rbf
>   File "C:\Program Files\Python39\lib\site-packages\scipy\interpolate\__init__.py", line 165, in <module>
>     from .interpolate import *
>   File "C:\Program Files\Python39\lib\site-packages\scipy\interpolate\interpolate.py", line 13, in <module>
>     import scipy.special as spec
>   File "C:\Program Files\Python39\lib\site-packages\scipy\special\__init__.py", line 633, in <module>
>     from . import _ufuncs
> ImportError: DLL load failed while importing _ufuncs: The specified module could not be found.

**Reply by slayoo** [reply]
Does the same thing happen if you just do ``from scipy.interpolate import Rbf``?
It seems that the SciPy installation is not functional...
And seems is not an uncommon problem:
- https://stackoverflow.com/questions/36394571/scipy-importerror-dll-load-failed-the-specified-module-could-not-be-found
- https://stackoverflow.com/questions/17480779/importerror-when-importing-certain-modules-from-scipy
Please do try one of the solutions suggested above?

**Reply by ghost** [reply]
Thanks Sylwester. You saved me several hours of googling. :)

**Reply by ghost** [reply]
Even though it works. It requires me open powershell with administrator access.

**Reply by slayoo** [reply]
What are the errors/symptoms if you run unprivileged?
